src/main/java/core/basesyntax/PurchaseOperation.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public class PurchaseOperation implements OperationHandler {
+    @Override
+    public void handle(FruitTransaction transaction, Storage storage) {

```

Comment:
Before removing fruits from the storage, you should check if the resulting balance after the purchase will be positive. If the balance is negative, throw a `RuntimeException` as specified in the checklist for handling the Purchase operation.